src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -416,37 +433,46 @@ private void ProcessProjectChange(Solution solution, ProjectId projectId)
                 return;
             }
 
-            installedPackages = new Dictionary<string, string>();
+            var installedPackages = new Dictionary<string, string>();
+            var isEnabled = false;
 
             // Calling into NuGet.  Assume they may fail for any reason.
             try
             {
                 var installedPackageMetadata = _packageServices.GetInstalledPackages(dteProject);
                 installedPackages.AddRange(installedPackageMetadata.Select(m => new KeyValuePair<string, string>(m.Id, m.VersionString)));
+                isEnabled = true;
+            }
+            catch (ArgumentException)
+            {
+                // Nuget may throw an ArgumentException when there is something about the project 
```

Comment:
It would be nice to have a comment about the issue we know about now, so that we can revisit removing this at that point.

Also, could we restrict further by doing `Message.Contains` or something horrible like that?